### DISCUSSION TITLE
Use of Vueform in Quasar

### DISCUSSION THREAD

**Discussion by erdesigns-eu** [vueform/vueform#120] [Questions] [answered]
I am trying to test Vueform with Quasar, but i run into some problems like errors about imports.
` SyntaxError: The requested module '/node_modules/wnumb/wNumb.js?v=a6e88e7a' does not provide an export named 'default' (at useTooltip.js:2:8)`

I see that this is one of the things that is used by some of the components in Vueform, in this case it is in the composable of the slider component. But i want to use quasar components, so i dont really need/want these components.

Is there a way to install or use Vueform without these default installed components? Or do i need to patch it manually to remove these? Any help is greatly appreciated.

**Comment by erdesigns-eu** [comment, marked as answer]
Okay a update here. I asked the same question on Stackoverflow and Discord and as someone pointed out to me i would need to install the Vite plugin. Now i tried to do this, but im getting errors about not being able to find the plugin.
I tried installing other vite plugins which work fine, so the problem is getting this plugin to install in Quasar CLI (which uses Vite under the hood).

Another update, installing the plugin doesnt seem to work. Getting import errors of all kinds, but copying the plugin content to the Quasar config does work. I updated the answer on stackoverflow with my config.
So maybe the developers can add support for use in Quasar with the Quasar CLI?

https://stackoverflow.com/questions/77826727/using-vueform-in-quasar/77830685#77830685

**Reply by adamberecz** [reply]
As of `1.7.0` Vueform should work without a Vite plugin. The only difference is that you have to import your theme file from `dist/`.

So replacing eg. in `vueform.config.js`:

```js
import vueform from '@vueform/vueform/themes/vueform'
```

with this:
```js
import vueform from '@vueform/vueform/dist/vueform'
```

should work.

**Reply by erdesigns-eu** [reply]
Thank you i will try this
